### PR TITLE
Automated type conversion

### PR DESCRIPTION
**Pull request by abrauchli**
This commit contains changes from the following commands:

    find . -type f \
        \( -name '*\.[ch]' -o -name '*\.cpp' -o -name '*\.ino' \) -print0 \
        | xargs -0 sed -i \
        -e 's/\bu\([0-9]\{1,2\}\)\b/uint\1_t/g' \
        -e 's/\bs\([0-9]\{1,2\}\)\b/int\1_t/g' \
        -e 's/\bf32\b/float32_t/g'
    make style-fix

This commit also updates embedded-common

### CODE REVIEW

**Comment by abrauchli** [comment]
(last commit is rebase on master, no changes)

File: sht-common/sht_common.c [hunk 1]
```diff
@@ -41,25 +41,25 @@
 #include "sensirion_i2c.h"
 #include "sht.h"
 
-s8 sht_common_read_ticks(u8 address, s32 *temperature_ticks,
-                         s32 *humidity_ticks) {
-    u8 data[6];
-    s8 ret = sensirion_i2c_read(address, data, sizeof(data));
+int8_t sht_common_read_ticks(uint8_t address, int32_t *temperature_ticks,
+                         int32_t *humidity_ticks) {
```
Comment (psachs) [inline sht-common/sht_common.c]: indent seems wrong here

File: sht-common/sht_common.h [hunk 2]
```diff
@@ -38,10 +38,10 @@
 extern "C" {
 #endif
 
-s8 sht_common_read_ticks(u8 address, s32 *temperature_ticks,
-                         s32 *humidity_ticks);
+int8_t sht_common_read_ticks(uint8_t address, int32_t *temperature_ticks,
+                         int32_t *humidity_ticks);
```
Comment (psachs) [inline sht-common/sht_common.h]: indent as well

File: sht-common/sht_common.c (hunk 1 above)
Comment (abrauchli) [reply sht-common/sht_common.c]: indeed

**Pull request merged**